graviti/dataframe/column/series.py

```diff
@@ -555,3 +573,78 @@ def __getitem__(self, key: int) -> Optional[FileBase]:
             return None
 
         return FileBase.from_pyarrow(scalar)
+
+
+@pt.ContainerRegister(pt.enum)
+class EnumSeries(Series):
+    """One-dimensional array for portex builtin type enum."""
+
+    _values: Dict[Optional[pa.Scalar], Union[int, float, str, bool, None]]
+
+    def __getitem__(self, key: int) -> Any:
+        return self._values[self._data[key]]
+
+    def _to_request_data(self, need_record_key: bool = True) -> List[int]:
+        return self._data.to_pyarrow().to_pylist()  # type: ignore[no-any-return]
+
+    @classmethod
+    def _from_factory(
+        cls: Type[_E],
+        factory: LazyFactoryBase,
+        schema: pt.PortexType,
+        root: Optional["DataFrame"] = None,
+        name: Tuple[str, ...] = (),
+    ) -> _E:
+        obj: _E = object.__new__(cls)
+        obj._data = factory.create_pyarrow_list()
+        obj.schema = schema
+        obj._root = root
+        obj._name = name
+        # pylint: disable=protected-access
+        obj._values = cls._get_enum_values(
+            schema.to_builtin().values, obj._data._patype  # type: ignore[attr-defined]
+        )
+
+        return obj
+
+    @classmethod
+    def _from_pyarrow(
+        cls: Type[_E],
+        array: pa.Array,
+        schema: pt.PortexType,
+        root: Optional["DataFrame"] = None,
+        name: Tuple[str, ...] = (),
+    ) -> _E:
+        obj: _E = object.__new__(cls)
+        obj._data = PyArrowPagingList.from_pyarrow(array)
+        obj.schema = schema
+        obj._root = root
+        obj._name = name
+        # pylint: disable=protected-access
+        obj._values = cls._get_enum_values(
+            schema.to_builtin().values, obj._data._patype  # type: ignore[attr-defined]
+        )
+        return obj
+
+    @staticmethod
+    def _get_enum_values(
```

Comment:
will `create` better than `get` here?

also, do we need a special name for the `values` here?